include/nil/crypto3/marshalling/zk/types/placeholder/common_data.hpp

```diff
@@ -140,10 +140,12 @@ namespace nil {
                 ){
                     auto fixed_values = make_commitment<Endianness, typename CommonDataType::commitment_scheme_type>(std::get<0>(filled_common_data.value()));
 
-                    typename CommonDataType::columns_rotations_type columns_rotations;
+                    typename CommonDataType::columns_rotations_type columns_rotations(
+                        std::get<1>(filled_common_data.value()).value().size()
```

Comment:
Hm, was this a bug? Looks like we were accessing out of the memory?

Reply:
`columns_rotations_type` was changed to no longer be an array (vector now), so we are passing the size to the constructor. This should not access out of memory; and address sanitizer isn't triggered